tools/shared/src/main/scala/org/scalajs/core/tools/javascript/JSDesugaring.scala

```diff
@@ -1199,6 +1228,21 @@ object JSDesugaring {
             }
           }
 
+        case JSBracketSuperCall(cls, receiver, method, args) =>
+          val superClass = getSuperClassOfJSClass(
+              classEmitter.linkedClassByName(cls.className))
+          val superCtor =
+            JSLoadConstructor(ClassType(superClass.encodedName)) // abuse
+
+          redo {
+            JSBracketMethodApply(
+                JSBracketSelect(
+                    JSBracketSelect(superCtor, StringLiteral("prototype")),
```

Comment:
It doesn't really matter. I used `JSBracketSelect` simply not to use `JSDotSelect`. I think in general `JSDotSelect` should be used only for things we can rename. And we cannot rename `prototype`.